src/include/duckdb/execution/operator/projection/physical_range.hpp

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/execution/operator/projection/physical_range.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/execution/physical_operator.hpp"
+#include "duckdb/planner/expression.hpp"
+
+namespace duckdb {
+
+//! PhysicalRange implements the physical RANGE operation on integers
+class PhysicalRange : public PhysicalOperator {
```

Comment:
As you mentioned, these are never instantiated so they shouldn't be PhysicalOperators.
As far as I can tell this is only used for the static GetState method.

Please rename the class, remove the boilerplate added to make this able to inherit from PhysicalOperator and move the file somewhere else.